test/oauth_proxy_outbound_test.js

```diff
@@ -21,7 +21,7 @@ describe('oauth_proxy outbound message integrity: verbs', function() {
     ['GET', 'DELETE'].forEach(function(verb) {
```

Comment:
Looks like it might be a node > 0.11.0 discrepancy: https://github.com/SamDecrock/node-http-ntlm/issues/21

You might need to check the node-http-proxy version and see if there's an upgrade to address this, assuming the issue is from that package.